configure.ac

```diff
@@ -47,10 +47,30 @@ AS_IF([test "x$found_introspection" = xyes],
 uname -p|grep s390
 on_s390=$?
 
```

Comment:
This configure check is broken. Running ```uname -p``` on own of our s390x builder boxes yields ```unknown```
I'd propose a patch like this instead using AC_CANONICAL_HOST
```
-uname -p|grep s390
-on_s390=$?
+AC_CANONICAL_HOST
»
-AM_CONDITIONAL(ON_S390, test "$on_s390" = "0")
+case $host_cpu in
+  s390|s390x)
+    AM_CONDITIONAL(ON_S390, true)
+    ;;
+esac
```

Reply:
Thanks, `AC_CANONICAL_HOST` is definitely better way to do this. Fixed.